docs/cli.md

```diff
@@ -522,51 +533,15 @@ gcli tag update --tag tagA --rename newTag
 gcli tag update --tag tagA --comment "new comment"
 ```
 
-### Role commands
+### Authentication
 
-#### Display role details
+#### Simple authentication
 
 ```bash
-gcli role details --role admin
-```
-
-#### List all roles
-
-```bash
-gcli role list
-```
-
-#### Create a role
-
-```bash
-gcli role create --role admin
-```
-
-#### Delete a role
+gcli <normal command> --simple
```

Comment:
?
```suggestion
gcli <normal command> --auth simple
```

Reply:
Not related to this PR. However, we could possibly do it that way, but as this is the only authentication type that will be directly supported via the command line as others will be supported via the configuration file.